src/win32/git2.rc

```diff
@@ -9,7 +9,18 @@
 # define LIBGIT2_COMMENTS "For more information visit http://libgit2.github.com/"
 #endif
 
+#ifdef GCC_WINDRES
+# define STRINGIZE(x) #x
+# define STRINGIZE_(x) STRINGIZE(x)
+#else
+# define STRINGIZE_(x) x
+#endif
+
+#ifdef GCC_WINDRES
+VS_VERSION_INFO		VERSIONINFO
```

Comment:
This is a good question 😄  These lines are the start of the structure. While other lines are indented, in fact they are indented because they are the fields of this structure. So they would have had the same indentation even if they haven't been inside `#if / #endif` blocks.

Let me see if it will even build if I indent them... But it will look a bit off anyway (compare with `BEGIN` and `END`, lines 35 and 55).

Reply:
No worries, I think this is fine.  😅